soc/arm/atmel_sam0/common/Kconfig.defconfig.series

```diff
@@ -0,0 +1,52 @@
+# Atmel SAM0 MCU family configuration options
+
+# Copyright (c) 2020 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_SAM0
+
+config ADC_SAM0
+	default y
```

Comment:
I think this would be cleaner if you did:

```
config ADC_SAM0
    default ADC
```

This means that `ADC_SAM0` inherits the `y/n` value from `ADC`.
Same for all others.

@MaureenHelm @galak Did we settle on using this idiom? I thought so at least

Reply:
Done